app/controllers/account/workspaces_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @workspace = resource
+    @members = User.for_workspace(@workspace).order_desc
```

Comment:
@workspace.potential_members (implement a new method)
it's more readable and better place to return possible members